install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -650,6 +650,19 @@ enableK8sEventHandover: false
 # -- Enable CiliumEndpointSlice feature.
 enableCiliumEndpointSlice: false
 
+envoyConfig:
+  # -- Enable envoy config CRD
```

Comment:
nit: add the full Kind, to make it easier to grep for.

Reply:
Done.